src/ameedf/jb8223/assignments/RotationCheck.java

```diff
@@ -42,6 +44,92 @@
  */
 public class RotationCheck {
     public static void main(String[] args) {
-        // TODO: enter your code here :)
+		Scanner in = new Scanner(System.in);
+		int firstNumber = 0, secondNumber = 0;
```

Comment:
you don't really need to initialize them because you are using a "do-while" loop

Reply:
Yes, i this case you are right. But I think, initialize declared variables - it's always good way. Theoretically, after time I can change the code and forgot to initialize variables.

Reply:
if you forget - Java is obligated to inform you (a compilation error)